.github/issue_template.md

```diff
@@ -0,0 +1,11 @@
+## Expected Behavior
```

Comment:
@glasnt Just pushed up an update, I did a squash & rebase to reduce commit history on this one. Thanks!